src/components/Filters/FilterChecklist.vue

```diff
@@ -1,147 +1,101 @@
 <template>
-  <div
-    :class="{ ['filters']: true, ['single']: isSingleFilter, 'mb-8': true }"
-    @click="hideLicenseExplanationVisibility()"
-    @keyup.enter="hideLicenseExplanationVisibility()"
-  >
+  <fieldset class="mb-8" @click.stop="hideLicenseExplanationVisibility">
+    <legend v-if="title" class="text-2xl font-semibold mb-2">
+      {{ title }}
+    </legend>
     <div
-      v-if="title"
-      class="filters-title"
-      @click.prevent="toggleFilterVisibility"
-      @keyup.enter="toggleFilterVisibility"
+      v-for="(item, index) in options"
+      :key="index"
+      class="flex justify-between items-center mt-3"
     >
-      <h4 class="filter-heading">{{ title }}</h4>
+      <VCheckbox
+        :id="item.code"
+        :key="index"
+        :checked="item.checked"
+        :name="itemName"
+        :value="item.code"
+        :disabled="isDisabled(item)"
+        @change="onValueChange"
+      >
+        <VLicense v-if="filterType === 'licenses'" :license="item.code" />
+        <template v-else>{{ itemLabel(item) }}</template>
+      </VCheckbox>
       <button
-        v-if="!filtersExpandedByDefault"
-        :aria-label="$t('filter-list.category-aria', { categoryName: title })"
-        class="filter-visibility-toggle is-white py-2"
+        v-if="filterType === 'licenses'"
+        :ref="`${item.code}licenseIcon`"
+        :aria-label="$t('browse-page.aria.license-explanation')"
+        class="appearance-none"
+        type="button"
+        @click.stop="toggleLicenseExplanationVisibility(item.code)"
```

Comment:
@sarayourfriend , There were two event handlers that got triggered when you clicked the License explanation button: one opened it, and the other closed it right away. Adding `.stop` to the `click` event fixed this.
There is still a problem of the explanation opening in weird places, that I think the `VPopover` could solve.

Reply:
Nice! Yeah using the popover component would be great here! We'd have to add support for the arrow though, I left that out cause there weren't any examples I found in the designs.

I think it would also fix another issue I saw where if you have a license explanation popover open and click on another license explanation button it only closes the existing one but doesn't open the one you clicked on. I think VPopover would fix this too.